azure-pipelines/azure-pr.yaml

```diff
@@ -0,0 +1,19 @@
+trigger: none
+pr: none
+jobs:
+  - job: AzureDevOpsPullRequestValidation
+    steps:
+    - task: TriggerBuild@4
+      inputs:
+        definitionIsInCurrentTeamProject: true
+        buildDefinition: '[flowey] hvlite PR'
+        queueBuildForUserThatTriggeredBuild: false
+        useSameBranch: false
+        branchToUse: 'main'
+        waitForQueuedBuildsToFinish: true
+        waitForQueuedBuildsToFinishRefreshTime: '30'
+        failTaskIfBuildsNotSuccessful: true
+        buildParameters: 'OssSubmoduleCommit: $(Build.SourceVersion)'
```

Comment:
Shouldn't we pass the current remote as well here ?

Reply:
I just updated the pipeline that we're calling to use this repo as the default.

Reply:
by this I mean github.com/microsoft/openvmm